### ISSUE TITLE
Help me get my client to look like this

### ISSUE THREAD

**Issue by donradzsk** [shalzuth/BraveHaxvius#93]
![pic](https://image.ibb.co/dUYpaK/34657184_d2aa5fa4_f3f1_11e7_9082_17b4e827698e_1.png)

@serenitygrant @OGVENDETTA @Cybotor @ggwpx @NeoTetsuya  @hantucloud @HotMoogle @lupier or anyone whose app has these options, how to get them? I already figured out the "do raid" part, the rest is hard.

**Comment by lupier** [comment]
You can modify app as you want. Open source code.

**Comment by donradzsk** [comment]
Right, I understand I can modify it. I just need the information from BraveExvius.cs and MainWindow.cs and any other files to get the options in the pic. That info isn't in the barebones app. Looking for any help before support for this completely dries up.

**Comment by donradzsk** [comment]
@hantucloud are you still working on this or playing this game?
